lib/mrsk/commands/healthcheck.rb

```diff
@@ -18,7 +20,7 @@ def run
   end
 
   def curl
-    [ :curl, "--silent", "--output", "/dev/null", "--write-out", "'%{http_code}'", "--max-time", "2", health_url ]
+    [ *config.healthcheck.fetch("curl"), *CURL_ARGS, health_url ]
```

Comment:
`curl` option now defaults to a `docker` invocation, limiting MRSK deps to Docker alone. No native curl install required.

Reply:
Love this.